File: Cafeteria.Shared/DTOs/FoodItemOrderDto.cs
namespace Cafeteria.Shared.DTOs;

public class FoodItemOrderDto
{
    public int Id { get; set; }
    public int OrderId { get; set; }
    public int StationId { get; set; }
    public int? SaleCardId { get; set; }
    public int? SaleSwipeId { get; set; }
    public int? SwipeCost { get; set; }
    public decimal? CardCost { get; set; }
    public bool Special { get; set; }
    public List<FoodItemOptionDto> Options { get; set; } = new();
}

File: ReceiptPrinter/DTOs/FoodItemOrderDto.py
from pydantic import BaseModel
from typing import Optional, List
from .FoodItemOptionDto import FoodItemOptionDto


class FoodItemOrderDto(BaseModel):
    Id: int
    OrderId: int
    StationId: int
    SaleCardId: Optional[int] = None
    SaleSwipeId: Optional[int] = None
    SwipeCost: Optional[int] = None
    CardCost: Optional[float] = None
    Special: bool
    Options: List[FoodItemOptionDto] = []
